internal/controller/auth.controller.go

```diff
@@ -170,3 +170,33 @@ func (ac *AuthController) ResetPassword(ctx *gin.Context) {
 
 	response.SuccessResponse(ctx, statusCode, "")
 }
+
+// @Summary ChangePassword
```

Comment:
Had run make swagger to update the swagger docs ?, I did not see any change in docs folder, should run make swagger to update swagger docs